cli/step_environment.go

```diff
@@ -0,0 +1,69 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/bitrise-io/bitrise/bitrise"
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/bitrise/tools"
+	envmanModels "github.com/bitrise-io/envman/models"
+)
+
+type prepareStepInputParams struct {
+	environment                 []envmanModels.EnvironmentItemModel
+	inputs                      []envmanModels.EnvironmentItemModel
+	buildRunResults             models.BuildRunResultsModel
+	inputEnvstorePath           string
+	isCIMode, isPullRequestMode bool
+}
+
+func prepareStepEnvironment(params prepareStepInputParams) ([]envmanModels.EnvironmentItemModel, error) {
```

Comment:
tools.EnvmanInitAtPath, tools.ExportEnvironmentsList and envmanModels.NewEnvJSONList is only used to feed bitrise.EvaluateTemplateToString's envList input, could not we create this list easier?

Reply:
We could, it is already done in https://github.com/bitrise-io/bitrise/pull/722